app/2.0/homepage/contact-card/contact-card.html

```diff
@@ -32,16 +32,18 @@
         display: none;
       }
     </style>
-    <content></content>
+    <slot></slot>
 
     <iron-icon icon="star" hidden$="{{!starred}}"></iron-icon>
   </template>
   <script>
-    Polymer({
```

Comment:
Hmm, this is code that's actually being run, right? In that case, the site is running on Polymer 1.0, which means we can't actually use a 2.0 element on that page :(